internal/impl/devbox.go

```diff
@@ -1207,14 +1207,33 @@ func (d *Devbox) Lockfile() *lock.File {
 }
 
 func (d *Devbox) RunXPaths() (string, error) {
+	runxBinPath := filepath.Join(d.projectDir, ".devbox", "virtenv", "runx", "bin")
+	if err := os.RemoveAll(runxBinPath); err != nil {
+		return "", err
+	}
+	if err := fileutil.EnsureDirExists(runxBinPath, 0o755, false); err != nil {
+		return "", err
+	}
 	packages := lo.Filter(d.InstallablePackages(), devpkg.IsRunX)
-	paths := []string{}
 	for _, pkg := range packages {
-		p, err := runx.Install(pkg.RunXPath())
+		paths, err := runx.Install(pkg.RunXPath())
 		if err != nil {
 			return "", err
 		}
-		paths = append(paths, p...)
+		for _, path := range paths {
+			// create symlink to all files in p
+			files, err := os.ReadDir(path)
```

Comment:
Won't this symlink all files in the archive, including non-binaries? I'm wondering if this will create a bunch of conflicts. For example, if multiple packages have a `README` in them.

Reply:
Yeah it will, but mostly for files we don't care about. We can fix this later on. (Requires priorities similar to nix)